### DISCUSSION TITLE
Is this 'COMBINING COMMA ABOVE' (U+0313)?

### DISCUSSION THREAD

**Discussion by jsbien** [psb1558/Junicode-font#95] [Q&A] [answered]
The same text as in my previous question, _aliter_ and _similaliter_ :
![Zaborowski1535p1l31_aliter1](https://user-images.githubusercontent.com/8024702/146632234-09b60d28-d495-4000-b4a6-c653e4910776.png)
![Zaborowski1535p3l28_similiter1](https://user-images.githubusercontent.com/8024702/146632243-d9a56141-a684-42d0-b9cb-af6fb472121a.png)

**Comment by psb1558** [comment, marked as answer]
These are variants of U+035B, combining zigzag above. MUFI has the relevant variants at U+F1C7 and U+F1C8 (I'd say use U+F1C8 for the one on top and U+F1C7 for the other, but that one could be interpreted either way). You can also get these via cv81, but that isn't yet complete.

There is a mad profusion of variants of this mark in MSS and early printed books, so many that one could hardly believe they are the same mark, but they are semantically identical.

**Reply by jsbien** [reply]
Thanks! The strange thing is the printer used two different characters in the same text...
